bin/load-mocks

```diff
@@ -449,15 +449,27 @@ def main(num_events=1, extra_events=False):
 
             raw_commits = generate_commits(user)
 
-            repo = Repository.objects.get_or_create(
-                organization_id=org.id,
-                provider='integrations:github',
-                external_id='example/example',
-                defaults={
-                    'name': 'Example Repo',
-                    'url': 'https://github.com/example/example',
-                }
-            )
+            try:
+                repo = Repository.objects.get_or_create(
+                    organization_id=org.id,
+                    provider='integrations:github',
+                    external_id='example/example',
+                    defaults={
+                        'name': 'Example Repo',
+                        'url': 'https://github.com/example/example',
+                    }
+                )
+            except IntegrityError:
+                # for users with legacy github plugin
+                # upgrade to the new integration
+                repo = Repository.objects.get(
+                    organization_id=org.id,
+                    provider='github',
+                    external_id='example/example',
+                    name='Example Repo',
+                )
+                repo.provider = 'integrations:github'
+                repo.save()
```

Comment:
The `repo` value is used as `repo[0].id` below. I don't think this variable matches the tuple that `get_or_create()` makes. You could unpack the results from `get_or_create()` and update the other uses of `repo`. That would let you use the bare model consistently.

Reply:
Wha.... why? oh my goodness why is it written this way. the other value isn't even used.... Thanks